pipelines/crc-qe-virtual-fedora.yaml

```
@@ -0,0 +1,199 @@
+          value: "false"
+        - name: download
+          value: "false"
+        - name: debug
+          value: "true"
+    - name: qe
+      runAfter:
+        - install-crc
+      taskRef:
+        resolver: git
+        params:
+        - name: url
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)